modules/store/schematics-core/utility/project.ts

```diff
@@ -52,3 +56,20 @@ export function isLib(
 
   return project.projectType === 'library';
 }
+
+export function getProjectMainFile(
```

Comment:
We keep our `schematics-core` synced across packages for consistency. This should be moved to

https://github.com/ngrx/platform/blob/master/modules/schematics-core/utility/project.ts

And then run 

```sh
yarn copy:schematics
```

To copy it to `schematics-core` folder in the other packages.
```

Reply:
Hm... I just tried to do the steps you described but it seems like the `yarn copy:schematics` does nothing in my case. I expect the code under `module/schematics-core` will be propagated to `store/schematics-core/...`, however, I see no changes after running this command. Am I missing something?
P.S. The output of `yarn copy:schematics` shows that the task was successfully executed.

Reply:
Hmm maybe it's an operating system path issue in our script. To not hold this up, if everything is good, we'll merge it and I'll follow up with the schematics-core move.

Reply:
Alright, so from my side no actions are needed anymore? :)

Reply:
Right :)